lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -257,7 +258,7 @@ def self.get_values(json, category_type, override_price = false)
       'cr1.8xlarge' => 240,
       'hs1.8xlarge' => 48000,
       'g2.2xlarge' => 60, 'g2.8xlarge' => 240,
-      'g3.4xlarge' => 0, 'g3.8xlarge' => 0, 'g3.16xlarge' => 0, # g3 are ebs-only
+      'g3.4xlarge' => 0, 'g3.8xlarge' => 0, 'g3.16xlarge' => 0, 'g3s.xlarge' => 0, # g3 are ebs-only
```

Comment:
why lump g3s in the same line instead of starting a new line
minor point, take it or leave it